platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -175,11 +184,53 @@ public void setUpTree() throws Exception {
     public void testInit() throws Exception {
         ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
                 testConfigAdminExt);
+        serviceBean.setMBeanServer(mBeanServer);
         serviceBean.init();
 
-        //Do it again, should remove and re-init
+        verify(mBeanServer).registerMBean(any(Object.class), any(ObjectName.class));
+    }
+
+    /**
+     * Tests the {@link ApplicationServiceBean#init()} method for the case
+     * where the serviceBean has already been initialized
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testInitTwice() throws Exception {
+        ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
+                testConfigAdminExt);
+        serviceBean.setMBeanServer(mBeanServer);
+        when(mBeanServer.registerMBean(any(Object.class), any(ObjectName.class)))
+                .thenThrow(new InstanceAlreadyExistsException()).thenAnswer(new Answer() {
```

Comment:
You should be able to use `thenReturn(null)` here instead of `thenAnswer()`. Also, shouldn't this be the other way around, return `null` first then throw an exception?